src/handbook/company/security/information-security-roles.md

```diff
@@ -0,0 +1,70 @@
+---
+navTitle: Information Security Roles and Responsibilities
+---
+
+# Information Security Roles and Responsibilities
+
+| Policy owner   | Effective date |
+| -------------- | -------------- |
+| @ZJvandeWeg    | 2023-06-01     |
+
+
+## Purpose
+
+FlowForge is committed to conducting business in compliance with all applicable laws,
+regulations, and company policies. FlowForge has adopted this policy to outline
+the security measures required to protect electronic information systems and
+related equipment from unauthorized use.
+
+## Roles and Responsibilities
+
+
+**Board of directors**
+
+ - Oversight over risk and internal control for information security, privacy, and compliance.
+ - Consults with executive leadership to understand FlowForge's security mission and risks and provides guidance to bring them into alignment
+
+
+**Executive Leadership**
+
+ - Approves Capital Expenditures for Information Security and Privacy programs and initiatives
+ - Oversight over the execution of the information security and Privacy risk management program and risk treatments
+ - Communication Path to the Board of Directors
+ - Aligns Information Security and Privacy Policy and Posture based on FlowForge's mission, strategic objectives and risk appetite
```

Comment:
Al
```suggestion
 - [Aligns Information Security](<insert link>) and [Privacy Policy](/privacy-policy/#privacy-policy) and [Posture](<link where?>) based on FlowForge's mission, strategic objectives and risk appetite
```

Reply:
From the template it isn't clear what Posture really means here. Have removed.
```suggestion
 - Aligns [Information Security](./information-security.md) and [Privacy Policy]((/privacy-policy/#privacy-policy) based on FlowForge's mission, strategic objectives and risk appetite
```